fdbclient/NativeAPI.actor.h

```diff
@@ -31,6 +31,7 @@
 #include "flow/flow.h"
 #include "flow/TDMetric.actor.h"
 #include "fdbclient/FDBTypes.h"
+#include "fdbclient/BlobGranuleCommon.h"
```

Comment:
double check all includes to avoid transitive ones

Reply:
I'll do this next